noir-projects/noir-contracts/contracts/static_parent_contract/src/main.nr

```diff
@@ -30,6 +30,16 @@ contract StaticParent {
         context.call_private_function(target_contract, target_selector, args).unpack_into()
     }
 
+    // Just like function above but with 3 args.
+    #[aztec(private)]
+    fn private_call_3_args(
```

Comment:
Needed to add this because now the private_call func was used with nested functions with different arg lengths.